google/cloud/security/scanner/audit/forwarding_rule_rules_engine.py

```diff
@@ -0,0 +1,253 @@
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules engine for Bucket acls"""
+from collections import namedtuple
+import re
+
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type.resource import ResourceType
+from google.cloud.security.common.util import log_util
+from google.cloud.security.scanner.audit import base_rules_engine as bre
+from google.cloud.security.scanner.audit import errors as audit_errors
+
+
+# TODO: The next editor must remove this disable and correct issues.
+# pylint: disable=missing-type-doc,missing-return-type-doc,missing-return-doc
+# pylint: disable=missing-param-doc,differing-param-doc
+
+
+LOGGER = log_util.get_logger(__name__)
+
+
+# TODO: move this to utils since it's used in more that one engine
+def escape_and_globify(pattern_string):
```

Comment:
This function was moved to https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/common/util/regex_util.py. Please use that one